index.js

```diff
@@ -221,9 +233,25 @@ module.exports = {
     return require('./lib/commands');
   },
 
+  treeForFastBoot: function(tree) {
+    // Instance initializer should only be included in build if required.
+    // It's only required for FastBoot support.
+    if (!this._needsFastBootSupport) {
+      tree = funnel(tree, {
```

Comment:
The tree argument passed in here is our own `./fastboot` folder, which should only contain the excluded file. I think we should change this to `return null` instead of excluding the specific file...

Reply:
Done so in 32e5d207490a6b5259b0454fdbf4b2790905b356.